project/blinky_interchange.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
What is the difference between this and the blinky.json project? I think that, if there is only the addition of the required_toolchain, this project is not actually needed, and the blinky one can be used instead